### PR TITLE
csharplang-attributes-1

### PR DESCRIPTION
**Pull request by chamukallem-msft**

### CODE REVIEW

File: spec/attributes.md [hunk 1]
```diff
@@ -162,84 +162,67 @@ The types of positional and named parameters for an attribute class are limited
...
-
-attribute_argument_expression
-    : expression
-    ;
+global_attributes: 
+    global_attribute_section+;
+
+global_attribute_section:
+     '[' global_attribute_target_specifier attribute_list ']' |
+     '[' global_attribute_target_specifier attribute_list ',' ']';
```
Comment (CyrusNajmabadi) [inline spec/attributes.md]: why was this changed?  The previous way is much easier to read.

Comment (CyrusNajmabadi) [reply spec/attributes.md]: specifically, having each production start with either `:` or `|` scans really well.  It's also how we generate our grammar files here: https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

**Comment by chamukallem-msft** [comment]
I should I have branched out of csharp5-standard
